homeassistant/components/london_underground/sensor.py

```diff
@@ -55,24 +65,43 @@ async def async_setup_platform(
     session = async_get_clientsession(hass)
 
     data = TubeData(session)
-    await data.update()
+    coordinator = LondonTubeCoordinator(hass, data)
+
+    await coordinator.async_config_entry_first_refresh()
 
     sensors = []
     for line in config[CONF_LINE]:
-        sensors.append(LondonTubeSensor(line, data))
+        sensors.append(LondonTubeSensor(coordinator, line))
 
     async_add_entities(sensors, True)
```

Comment:
We don't need to request a refresh again from the coordinator during entity addition.
```suggestion
    async_add_entities(sensors)
```